mozilla-lockwise/mozilla-lockwise.github.io
# Add top bar for Join Firefox

<img width="1552" alt="image" src="https://user-images.githubusercontent.com/49511/58710602-ebb06c00-8379-11e9-8cb4-36b67754fc5b.png">

![Screen Shot 2019-05-31 at 07 58 55](https://user-images.githubusercontent.com/49511/58710622-fb2fb500-8379-11e9-94b2-427f4eef1ca5.png)



- [x] add banner to home page (only) with tracking link
- [x] fix bug with flex re-calculating the Firefox logo instead of keeping it 24px on small responsive layout
- [x] change from a fixed position over the page to actually block content pushing the page content down
- [x] make sure "Join Firefox" wraps together onto new line Fixes #130

---

tldr on how to test locally, see readme for more details:

- clone this repo
- check out this branch
- run the `bundle exec jekyll serve` commands
